tests/end2end/test_nodecellar.py

```diff
@@ -40,3 +40,21 @@ def _verify_deployed_service_in_storage(service_name, model_storage):
     assert service.name == service_name
     assert len(service.executions) == 0  # dry executions leave no traces
     assert len(service.nodes) == 10
+    assert set(service.nodes['loadbalancer_host_1'].interfaces['Standard'].operations['create'] \
```

Comment:
While these additions are fine, I think this JIRA really calls for unit tests which will check the various options for passing in arguments and verify everything overrides what it should and only that.
There are many such scenarios, and you don't want to test them all in e2e tests.